### DISCUSSION TITLE
[NO Longer Bountied] Somehow porting Citadel-style chemistry

### DISCUSSION THREAD

**Discussion by ReturnToZender** [Bubberstation/Bubberstation#398] [ideas/bounties]
Citadel chemistry is good. It's popular, simple to understand, and minimizes spreadsheeting necessary.

Sure, it will have powergaming, but it will overall be a good change for the server. We can fix what we break after.

Citadel-style is specifically referencing the chemistry system available on Citadel Main, here: 
https://github.com/Citadel-Station-13/Citadel-Station-13
This is referred to as "Trekchem," and revolves around Bicaridine, Kelotane, Tricordrazine, and Anti-Tox (please rename it to Dylovene)

**Comment by DerFlammenwerfer** [comment]
The easiest way to implement simpler chem production that doesn't need micromanagement is to add the recipes for reagents you want to make without using the cat_2 variables for those chems. The basic healing chems currently use cat2_medicines.dm for their definitions, so you could re-add the trekchems just as simply as copying and pasting the recipes for them from Citadel. The recipe system trekchems are produced under is the same old system that all the drink recipes are made with; they're cross-compatible, as technically you could make the trekchems into cat2 chems just by adding the variables that affect purity such as exo/endothermic reactions, pH changes, etc.

**Comment by DerFlammenwerfer** [comment]
Turns out it's not as simple as I thought it would be, but that's okay. I'm working on it.
